Set up a scheduled backup by default (#5151)
kyma-project/kyma: docs/backup/05-01-velero.md

| Parameter | Description | Default value |
|-----------|-------------|---------------|

> | **schedules** | Sets up a scheduled backup | A scheduled backup runs at 07:00 on every day-of-week from Monday through Friday. |

Reviewer: Maybe here we should also add the couple of values that can be set to change the scheduled backup, like `schedules.scheduled-backup.schedule` and `schedules.scheduled-backup.template.ttl`.

> | **schedules** | Sets up a scheduled backup | A scheduled backup runs at 07:00 on every day-of-week from Monday through Friday. |

Author: Done

> | **schedules** | Sets up a scheduled backup. By default, a scheduled backup runs at 07:00 on every day-of-week from Monday through Friday. | scheduled-backup |

Reviewer 2: ```suggestion
| **schedules** | Sets up a scheduled backup. By default, a scheduled backup runs at 07:00 daily on Monday through Friday. | scheduled-backup |
```

Reviewer 2: If scheduled-backup is the default value then you should format it as `code`.
